packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -343,12 +385,23 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
       let item = renderedItems[i];
       let itemWidth = this._commandItemWidths![item.key];
 
+      if (!itemWidth) {
+        continue;
+      }
+
       if ((consumedWidth + itemWidth) >= availableWidth) {
         if (i > 0 && !isOverflowVisible && (availableWidth - consumedWidth) < OVERFLOW_WIDTH) {
           i--;
         }
 
-        renderedOverflowItems = renderedItems.splice(i).concat(overflowItems!);
+        // Insert divider between overflow items
+        renderedOverflowItems = renderedItems
+          .splice(i)
+          .concat({
```

Comment:
Why are you adding a new divider here?

Reply:
I think it looks nice, but since I'm not a designer, I guess this isn't really up to me.